hudi-common/src/main/java/org/apache/hudi/common/table/read/UnmergedFileGroupRecordBuffer.java

```diff
@@ -52,9 +51,8 @@ public UnmergedFileGroupRecordBuffer(
       RecordMergeMode recordMergeMode,
       PartialUpdateMode partialUpdateMode,
       TypedProperties props,
-      HoodieReadStats readStats,
-      boolean emitDelete) {
-    super(readerContext, hoodieTableMetaClient, recordMergeMode, partialUpdateMode, props, readStats, Option.empty(), emitDelete);
+      HoodieReadStats readStats) {
+    super(readerContext, hoodieTableMetaClient, recordMergeMode, partialUpdateMode, props, readStats, Option.empty(), null);
```

Comment:
Does passing `null` as the update processor work here, or `UnmergedFileGroupRecordBuffer` is not really used so we can get away for now?

Reply:
The `UnmergedFileGroupRecordBuffer` is used but it overrides the methods that use the update processor

Reply:
yes, unmerged read is not related with cdc logging.

Reply:
Makes sense